docs/RemovingPresets.md

```diff
@@ -0,0 +1,91 @@
+[Readme](README.md) » Removing Presets
+
+## Removing Presets
+
+1. [Introduction](#introduction)
+2. [FSXA-UI](#fsxa-ui)
+3. [TailwindCSS](#tailwindcss)
+4. [Vue-TSX-Support](#vue-tsx-support)
+5. [Semantic-Release](#semantic-release)
+
+### Introduction
+
+This chapter introduces optional dependencies. Why we have them in our project and how you can easily remove them.
+### FSXA-UI
+
+The [FSXA-UI](https://github.com/e-Spirit/fsxa-ui) is a component library containing all sections that have been implemented in our project 'Smart Living'.
+
+#### Why do we use this?
+
+We use this component library to display the individual components in the PWA.
+We do not have to implement them here, but provide them with a detailed documentation.
+
```

Comment:
The second sentence is not clear to me. Does it mean we can implement components elsewhere (like in the pwa project directly), but still can/ 'have to' document them in this project? That seems strange to me.

Reply:
I must agree with you, but does not have an idea how to write this better. Let's talk about this.